.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: pnpm/action-setup@v2.2.2
+      - uses: pnpm/action-setup@v2.4.0
         with:
           version: 7.8.0
```

Comment:
pnpm version is wrong, it causes CI to fail. also we shouldn't specify it and instead load it from `"packageManager"` via corepack